packages/firestore/test/unit/local/local_store.test.ts

```diff
@@ -53,7 +53,7 @@ import {
   WatchTargetChange,
   WatchTargetChangeState
 } from '../../../src/remote/watch_change';
-import { assert } from '../../../src/util/assert';
+import { softAssert } from '../../../src/util/assert';
```

Comment:
The tests themselves are actually never minified. We run the full tests against the minified build, so even in this mode the "softAsserts" are kept. I think the new name "debugAsserts" conveys that they are run in tests better than "softAsserts" and almost as well as "hardAsserts", so I kept it as for now. I can go either way here...

FWIW, rather than changing these to hardAsserts, we might even want to use failed test expectations.

Reply:
Makes sense.

Changing to failed test expectations would be fine too, but seems orthogonal to the issue at hand and seems like it could be deferred.